website/content/docs/enterprise/license/manual-reporting.mdx

```diff
@@ -0,0 +1,166 @@
+---
+layout: docs
+page_title: Manual license utilization reporting
+description: >-
+  Manual license utilization reporting allows you to export, review, and send license utilization data to HashiCorp through the CLI or HCP Web Portal.
+---
+
+# Manual license utilization reporting
+
+@include 'alerts/enterprise-only.mdx'
+
+## How to manually send data reports
+
+### Generate a data bundle
+
+Data bundles include collections of JSON snapshots that contain license
+utilization information. This data will not be transmitted to HashiCorp
+automatically if manual reporting is enabled
```

Comment:
Manual reporting will always be enabled. Automatic reporting can be disabled but it is also enabled by default. If automatic reporting is enabled, data will be transmitted to Hashicorp directly. If not, customers should generate reports using the CLI command and then upload the report to Hashicorp.